src/test/java/uk/gov/hmcts/reform/pip/subscription/management/controllers/SubscriptionControllerTest.java

```diff
@@ -62,18 +65,45 @@ class SubscriptionControllerTest {
 
     UserSubscription userSubscription;
 
+    SubscriptionListType subscriptionListType;
+
     @BeforeEach
     void setup() {
-        mockSubscription = SubscriptionUtils.createMockSubscription(USER_ID, SEARCH_VALUE, EMAIL, LocalDateTime.now(),
-                                                                    ListType.CIVIL_DAILY_CAUSE_LIST
-        );
+        mockSubscription = SubscriptionUtils.createMockSubscription(USER_ID, SEARCH_VALUE, EMAIL, LocalDateTime.now());
         userSubscription = new UserSubscription();
+        subscriptionListType = new SubscriptionListType(USER_ID, LIST_TYPES, LIST_LANGUAGE);
+
+    }
+
+    @Test
+    void testCreateCourtSubscription() {
```

Comment:
Only minor, but could we change this to testCreateLocationSubscription

Reply:
done